DisCatSharp.ApplicationCommands/Checks/ApplicationCommandEqualityChecks.cs

```diff
@@ -70,13 +70,15 @@ internal static bool SoftEqual(this DiscordApplicationCommand source, DiscordApp
 					ApplicationCommandType.ChatInput => DeepEqual(source, target, localizationEnabled),
 					_ => (source.Name == target.Name)
 						&& (source.Type == target.Type) && (source.NameLocalizations == target.NameLocalizations)
-						&& (source.DefaultMemberPermissions == target.DefaultMemberPermissions) && (source.DmPermission == target.DmPermission)
```

Comment:
Note on this: Context menu commands can't have dm permission because they aren't meant for dms.